src/languages/en.ts

```diff
@@ -2439,24 +2455,13 @@ export default {
                 [CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE]: 'Not imported',
                 [CONST.INTEGRATION_ENTITY_MAP_TYPES.REPORT_FIELD]: 'Imported, displayed as report fields',
             },
-            disconnectPrompt: (integrationToConnect?: ConnectionName, currentIntegration?: ConnectionName): string => {
-                switch (integrationToConnect) {
-                    case CONST.POLICY.CONNECTIONS.NAME.QBO:
-                        return 'Are you sure you want to disconnect Xero to set up QuickBooks Online?';
-                    case CONST.POLICY.CONNECTIONS.NAME.XERO:
-                        return 'Are you sure you want to disconnect QuickBooks Online to set up Xero?';
-                    default: {
-                        switch (currentIntegration) {
-                            case CONST.POLICY.CONNECTIONS.NAME.QBO:
-                                return 'Are you sure you want to disconnect QuickBooks Online?';
-                            case CONST.POLICY.CONNECTIONS.NAME.XERO:
-                                return 'Are you sure you want to disconnect Xero?';
-                            default: {
-                                return 'Are you sure you want to disconnect this integration?';
-                            }
-                        }
-                    }
+            disconnectPrompt: (currentIntegration?: string, integrationToConnect?: string): string => {
+                if (!integrationToConnect) {
```

Comment:
@SzymczakJ @hungvu193 @yuwenmemon 

Did we decide to update these copies? I just added a [PR](https://github.com/Expensify/App/pull/44218/commits/c30025725e95b1c29c14ba4e14ae14c8b7c7e033) to fix these hardcoded values.


Also raised it on Slack [thread](https://expensify.slack.com/archives/C036QM0SLJK/p1719106879986199)